plan.go

```diff
@@ -69,7 +69,7 @@ func Plan(ctx *PlanningContext) (*queryPlan, error) {
 	}, nil
 }
 
-func createSteps(ctx *PlanningContext, insertionPoint []string, parentType, parentLocation string, selectionSet ast.SelectionSet) ([]*queryPlanStep, error) {
+func createSteps(ctx *PlanningContext, insertionPoint []string, parentType, parentLocation string, selectionSet ast.SelectionSet, substep bool) ([]*queryPlanStep, error) {
```

Comment:
```suggestion
func createSteps(ctx *PlanningContext, insertionPoint []string, parentType, parentLocation string, selectionSet ast.SelectionSet, childstep bool) ([]*queryPlanStep, error) {
```
Use the same name as the other function?